.github/workflows/test.yml

```diff
@@ -71,3 +71,99 @@ jobs:
             Meadowlark-js/integration-tests/junit.xml
             Meadowlark-js/system-tests/junit.xml
           retention-days: 1
+
+  end-to-end:
+    name: End to End tests for ${{ matrix.config.db }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          [
+            { db: "Mongo", plugin: "@edfi/meadowlark-mongodb-backend" },
+            { db: "PGSQL", plugin: "@edfi/meadowlark-postgresql-backend" },
+          ]
+
+    defaults:
+      run:
+        working-directory: Meadowlark-js
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v3.0.2
+
+      - name: NPM caching
+        uses: actions/setup-node@17f8bd926464a1afa4c6a11669539e9c1ba77048 # v3.2.0
+        with:
+          node-version: "16"
+          cache: "yarn"
+          cache-dependency-path: "**/yarn.lock"
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Start OpenSearch Container
+        run: docker compose -f ./backends/meadowlark-opensearch-backend/docker/docker-compose.yml up -d
```

Comment:
In a future story, we should experiment with using [service containers](https://docs.github.com/en/actions/using-containerized-services/about-service-containers) to see (a) if they have any performance benefits and (b) which use cases they might help with. For (b), I don't know, for example, if there is an available OpenSearch container that will run as a service easily.

Frankly, at some point I forgot about service containers, and the pattern seen here has been used in several places already. That's why I'm not concerned about changing it right now. Just need to investigate.